Cargo.toml

```diff
@@ -47,8 +47,7 @@ blake2 = { version = "0.9", optional = true }
 rand = { version = "0.8", optional = true }
 sha2 = { version = "0.9", optional = true }
 x25519-dalek = { version = "1.1", optional = true }
-pqcrypto-kyber = { version = "0.6", optional = true }
-pqcrypto-traits = { version = "0.3", optional = true }
+oqs = { git = "https://github.com/open-quantum-safe/liboqs-rust.git", rev = "6653b262d8548a8c93000a5b847b65c9b51e6e78", optional = true, default-features = false }
```

Comment:
This is dangerous, in the context of publishing on <https://crates.io>. crates.io does not allow dependencies that are pulled from a Git repo. (See also [RFC 2141](https://rust-lang.github.io/rfcs/2141-alternative-registries.html).) There has to be a crate here.

Reply:
Right. Can we convince the maintainers of the oqs crate to adopt a reasonable versioning scheme which progresses faster than every 6 months?

Reply:
I'd really rather not have to fork oqs and fork snow. But the way things are now it's looking like I'm going to have to in order to actually do a "cargo publish". So sad.